### DISCUSSION TITLE
Welcome to Fermentrack Discussions!

### DISCUSSION THREAD

**Discussion by thorrak** [thorrak/fermentrack#549] [General]
## 👋 Welcome!

For bugs, feature requests, or concerns there are GitHub issues -- for everything else, there's GitHub Discussions.

Although the majority of the discussion around Fermentrack takes place on the Homebrewtalk forums, not everyone is a member over there. If you have a question, loose idea for a feature, or just want to post to show off your build, feel free to do it here!

**Comment by Garzlok** [comment]
...I’m still waiting for someone on HBT asking if Fermentrack is compatible with an Atari 2600 or Commodore64

**Comment by uSlackr** [comment]
Well? Is it? 😀

\\\\ Greg
________________________________

**Comment by lbussy** [comment]
It's compatible with the most current versions of the TRS 80 operating system.  The C64 will be after Docker.

**Comment by thorrak** [comment]
...excuse me while I run off to see if docker runs on a C64.

**Comment by uSlackr** [comment]
Funny.


\\Greg

**Comment by b-a-p** [comment]
Hello!
I'd like to know if in near future documentation will be added for missing parts, currently I'm interested in connecting WiFi enabled hydrometrs (I'm using rapt but I believe process is more or less is universal).

**Reply by thorrak** [reply]
There is no standard for digital hydrometers, unfortunately, so adding them requires developing support for each individual class of device. People have requested RAPT support previously - here is the GitHub issue that tracks the status: https://github.com/thorrak/fermentrack/issues/674
